.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2007-present the original author or authors.
```

Comment:
I don't think this needs to be checked in. The webservice repo should have the minimum needed